### PR TITLE
[Outlook] Remove implication item id is getCallbackTokenAsync param

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[edc19f7](https://github.com/OfficeDev/office-js-docs-pr/commits/edc19f7a3c8f2fac73de57f9f5b823d1ca10d469)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910161809094358-1370%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=dSn8mrOHGGDX%2F%2BNwP5KAJ6Ffn5b9Gc9arxl2eydeAtA%3D&st=2019-10-16T18%3A07%3A45Z&se=2019-11-16T18%3A12%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[881c11e](https://github.com/OfficeDev/office-js-docs-pr/commits/881c11eb3a26b02c82ec973e4af758e10f6f7676)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910162030409918-1370%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=USKP6lORDelmxjjiPWS%2BenZ6yGw55xi%2BIOvKpWV%2BghI%3D&st=2019-10-16T20%3A29%3A03Z&se=2019-11-16T20%3A34%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ca6e25f](https://github.com/OfficeDev/office-js-docs-pr/commits/ca6e25fe86c53f248c8e7bebe06a26aaad09bb33)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910162316264842-1370%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=KrT7WJzg4ANoFbQMDNUKjiOr37btveQU0tUJmB5ctAk%3D&st=2019-10-16T23%3A12%3A38Z&se=2019-11-16T23%3A17%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.md [hunk 1]
```diff
@@ -491,6 +495,8 @@ When an EWS token is requested (`options.isRest = false`), the resulting token w
 
 The add-in should use the `ewsUrl` property to determine the correct URL to use when making EWS calls.
 
+You can pass the token and an attachment identifier or item identifier to a third-party system. The third-party system uses the token as a bearer authorization token to call the Exchange Web Services (EWS) [GetAttachment](/exchange/client-developer/web-service-reference/getattachment-operation) or [GetItem](/exchange/client-developer/web-service-reference/getitem-operation) operation to return an attachment or item. For example, you can create a remote service to [get attachments from the selected item](/outlook/add-ins/get-attachments-of-an-outlook-item).
```
Comment (kbrandl) [inline docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.md]:
```suggestion
You can pass the token and an attachment identifier or item identifier to a third-party system. The third-party system uses the token as a bearer authorization token to call the Exchange Web Services (EWS) [GetAttachment](/exchange/client-developer/web-service-reference/getattachment-operation) operation or [GetItem](/exchange/client-developer/web-service-reference/getitem-operation) operation to return an attachment or item. For example, you can create a remote service to [get attachments from the selected item](/outlook/add-ins/get-attachments-of-an-outlook-item).
```

File: docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.md [hunk 2]
```diff
@@ -546,9 +552,9 @@ The `getCallbackTokenAsync` method makes an asynchronous call to get an opaque t
 
 You can pass the token and an attachment identifier or item identifier to a third-party system. The third-party system uses the token as a bearer authorization token to call the Exchange Web Services (EWS) [GetAttachment](/exchange/client-developer/web-service-reference/getattachment-operation) or [GetItem](/exchange/client-developer/web-service-reference/getitem-operation) operation to return an attachment or item. For example, you can create a remote service to [get attachments from the selected item](/outlook/add-ins/get-attachments-of-an-outlook-item).
 
-Your app must have the **ReadItem** permission specified in its manifest to call the `getCallbackTokenAsync` method in read mode.
+To call `getCallbackTokenAsync` in read mode, **ReadItem** is the minimum permission level your app can specify in its manifest.
```
Comment (kbrandl) [inline docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.md]:
```suggestion
To call the `getCallbackTokenAsync` method in read mode, **ReadItem** is the minimum permission level your app can specify in its manifest.
```

File: docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.md [hunk 3]
```diff
@@ -546,9 +552,9 @@ The `getCallbackTokenAsync` method makes an asynchronous call to get an opaque t
...
 
-In compose mode you must call the [`saveAsync`](Office.context.mailbox.item.md#saveasyncoptions-callback) method to get an item identifier to pass to the `getCallbackTokenAsync` method. Your app must have **ReadWriteItem** permissions to call the `saveAsync` method.
+If the item has not been saved yet in compose mode, you must call the [`saveAsync`](Office.context.mailbox.item.md#saveasyncoptions-callback) method before calling the `getCallbackTokenAsync` method. To call the `saveAsync` method, **ReadWriteItem** is the minimum permission level your app can specify.
```
Comment (kbrandl) [inline docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.md]:
```suggestion
If the item has not yet been saved in compose mode, you must call the [`saveAsync`](Office.context.mailbox.item.md#saveasyncoptions-callback) method before calling the `getCallbackTokenAsync` method. To call the `saveAsync` method, **ReadWriteItem** is the minimum permission level your app can specify in its manifest.
```

File: docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md [hunk 4]
```diff
@@ -540,7 +540,11 @@ Gets a string that contains a token used to call REST APIs or Exchange Web Servi
 The `getCallbackTokenAsync` method makes an asynchronous call to get an opaque token from the Exchange Server that hosts the user's mailbox. The lifetime of the callback token is 5 minutes.
 
 > [!NOTE]
-> It is recommended that add-ins use the REST APIs instead of Exchange Web Services whenever possible. 
+> It is recommended that add-ins use the REST APIs instead of Exchange Web Services whenever possible.
+
+To call `getCallbackTokenAsync` in read mode, **ReadItem** is the minimum permission level your app can specify in its manifest.
```
Comment (kbrandl) [inline docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md]:
```suggestion
To call the `getCallbackTokenAsync` method in read mode, **ReadItem** is the minimum permission level your app can specify in its manifest.
```

File: docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md [hunk 5]
```diff
@@ -540,7 +540,11 @@ Gets a string that contains a token used to call REST APIs or Exchange Web Servi
...
+
+If the item has not been saved yet in compose mode, you must call the [`saveAsync`](Office.context.mailbox.item.md#saveasyncoptions-callback) method before calling the `getCallbackTokenAsync` method. To call the `saveAsync` method, **ReadWriteItem** is the minimum permission level your app can specify.
```
Comment (kbrandl) [inline docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md]:
```suggestion
If the item has not yet been saved in compose mode, you must call the [`saveAsync`](Office.context.mailbox.item.md#saveasyncoptions-callback) method before calling the `getCallbackTokenAsync` method. To call the `saveAsync` method, **ReadWriteItem** is the minimum permission level your app can specify in its manifest.
```

File: docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md [hunk 6]
```diff
@@ -554,6 +558,8 @@ When an EWS token is requested (`options.isRest = false`), the resulting token w
 
 The add-in should use the `ewsUrl` property to determine the correct URL to use when making EWS calls.
 
+You can pass the token and an attachment identifier or item identifier to a third-party system. The third-party system uses the token as a bearer authorization token to call the Exchange Web Services (EWS) [GetAttachment](/exchange/client-developer/web-service-reference/getattachment-operation) or [GetItem](/exchange/client-developer/web-service-reference/getitem-operation) operation to return an attachment or item. For example, you can create a remote service to [get attachments from the selected item](/outlook/add-ins/get-attachments-of-an-outlook-item).
```
Comment (kbrandl) [inline docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md]:
```suggestion
You can pass the token and an attachment identifier or item identifier to a third-party system. The third-party system uses the token as a bearer authorization token to call the Exchange Web Services (EWS) [GetAttachment](/exchange/client-developer/web-service-reference/getattachment-operation) operation or [GetItem](/exchange/client-developer/web-service-reference/getitem-operation) operation to return an attachment or item. For example, you can create a remote service to [get attachments from the selected item](/outlook/add-ins/get-attachments-of-an-outlook-item).
```

File: docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md [hunk 7]
```diff
@@ -609,9 +615,9 @@ The `getCallbackTokenAsync` method makes an asynchronous call to get an opaque t
 
 You can pass the token and an attachment identifier or item identifier to a third-party system. The third-party system uses the token as a bearer authorization token to call the Exchange Web Services (EWS) [GetAttachment](/exchange/client-developer/web-service-reference/getattachment-operation) or [GetItem](/exchange/client-developer/web-service-reference/getitem-operation) operation to return an attachment or item. For example, you can create a remote service to [get attachments from the selected item](/outlook/add-ins/get-attachments-of-an-outlook-item).
 
-Your app must have the **ReadItem** permission specified in its manifest to call the `getCallbackTokenAsync` method in read mode.
+To call `getCallbackTokenAsync` in read mode, **ReadItem** is the minimum permission level your app can specify in its manifest.
```
Comment (kbrandl) [inline docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md]:
```suggestion
To call the `getCallbackTokenAsync` method in read mode, **ReadItem** is the minimum permission level your app can specify in its manifest.
```

File: docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md [hunk 8]
```diff
@@ -609,9 +615,9 @@ The `getCallbackTokenAsync` method makes an asynchronous call to get an opaque t
...
 
-In compose mode you must call the [`saveAsync`](Office.context.mailbox.item.md#saveasyncoptions-callback) method to get an item identifier to pass to the `getCallbackTokenAsync` method. Your app must have **ReadWriteItem** permissions to call the `saveAsync` method.
+If the item has not been saved yet in compose mode, you must call the [`saveAsync`](Office.context.mailbox.item.md#saveasyncoptions-callback) method before calling the `getCallbackTokenAsync` method. To call the `saveAsync` method, **ReadWriteItem** is the minimum permission level your app can specify.
```
Comment (kbrandl) [inline docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md]:
```suggestion
If the item has not yet been saved in compose mode, you must call the [`saveAsync`](Office.context.mailbox.item.md#saveasyncoptions-callback) method before calling the `getCallbackTokenAsync` method. To call the `saveAsync` method, **ReadWriteItem** is the minimum permission level your app can specify in its manifest.
```

File: docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md [hunk 9]
```diff
@@ -543,6 +543,10 @@ The `getCallbackTokenAsync` method makes an asynchronous call to get an opaque t
 > [!NOTE]
 > It is recommended that add-ins use the REST APIs instead of Exchange Web Services whenever possible.
 
+To call `getCallbackTokenAsync` in read mode, **ReadItem** is the minimum permission level your app can specify in its manifest.
```
Comment (kbrandl) [inline docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md]:
```suggestion
To call the `getCallbackTokenAsync` method in read mode, **ReadItem** is the minimum permission level your app can specify in its manifest.
```

File: docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md [hunk 10]
```diff
@@ -543,6 +543,10 @@ The `getCallbackTokenAsync` method makes an asynchronous call to get an opaque t
...
+
+If the item has not been saved yet in compose mode, you must call the [`saveAsync`](Office.context.mailbox.item.md#saveasyncoptions-callback) method before calling the `getCallbackTokenAsync` method. To call the `saveAsync` method, **ReadWriteItem** is the minimum permission level your app can specify.
```
Comment (kbrandl) [inline docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md]:
```suggestion
If the item has not yet been saved in compose mode, you must call the [`saveAsync`](Office.context.mailbox.item.md#saveasyncoptions-callback) method before calling the `getCallbackTokenAsync` method. To call the `saveAsync` method, **ReadWriteItem** is the minimum permission level your app can specify in its manifest.
```

File: docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md [hunk 11]
```diff
@@ -555,6 +559,8 @@ When an EWS token is requested (`options.isRest = false`), the resulting token w
 
 The add-in should use the `ewsUrl` property to determine the correct URL to use when making EWS calls.
 
+You can pass the token and an attachment identifier or item identifier to a third-party system. The third-party system uses the token as a bearer authorization token to call the Exchange Web Services (EWS) [GetAttachment](/exchange/client-developer/web-service-reference/getattachment-operation) or [GetItem](/exchange/client-developer/web-service-reference/getitem-operation) operation to return an attachment or item. For example, you can create a remote service to [get attachments from the selected item](/outlook/add-ins/get-attachments-of-an-outlook-item).
```
Comment (kbrandl) [inline docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md]:
```suggestion
You can pass the token and an attachment identifier or item identifier to a third-party system. The third-party system uses the token as a bearer authorization token to call the Exchange Web Services (EWS) [GetAttachment](/exchange/client-developer/web-service-reference/getattachment-operation) operation or [GetItem](/exchange/client-developer/web-service-reference/getitem-operation) operation to return an attachment or item. For example, you can create a remote service to [get attachments from the selected item](/outlook/add-ins/get-attachments-of-an-outlook-item).
```

File: docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md [hunk 12]
```diff
@@ -610,9 +616,9 @@ The `getCallbackTokenAsync` method makes an asynchronous call to get an opaque t
 
 You can pass the token and an attachment identifier or item identifier to a third-party system. The third-party system uses the token as a bearer authorization token to call the Exchange Web Services (EWS) [GetAttachment](/exchange/client-developer/web-service-reference/getattachment-operation) or [GetItem](/exchange/client-developer/web-service-reference/getitem-operation) operation to return an attachment or item. For example, you can create a remote service to [get attachments from the selected item](/outlook/add-ins/get-attachments-of-an-outlook-item).
 
-Your app must have the **ReadItem** permission specified in its manifest to call the `getCallbackTokenAsync` method in read mode.
+To call `getCallbackTokenAsync` in read mode, **ReadItem** is the minimum permission level your app can specify in its manifest.
```
Comment (kbrandl) [inline docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md]:
```suggestion
To call the `getCallbackTokenAsync` method in read mode, **ReadItem** is the minimum permission level your app can specify in its manifest.
```

File: docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md [hunk 13]
```diff
@@ -610,9 +616,9 @@ The `getCallbackTokenAsync` method makes an asynchronous call to get an opaque t
...
 
-In compose mode you must call the [`saveAsync`](Office.context.mailbox.item.md#saveasyncoptions-callback) method to get an item identifier to pass to the `getCallbackTokenAsync` method. Your app must have **ReadWriteItem** permissions to call the `saveAsync` method.
+If the item has not been saved yet in compose mode, you must call the [`saveAsync`](Office.context.mailbox.item.md#saveasyncoptions-callback) method before calling the `getCallbackTokenAsync` method. To call the `saveAsync` method, **ReadWriteItem** is the minimum permission level your app can specify.
```
Comment (kbrandl) [inline docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md]:
```suggestion
If the item has not yet been saved in compose mode, you must call the [`saveAsync`](Office.context.mailbox.item.md#saveasyncoptions-callback) method before calling the `getCallbackTokenAsync` method. To call the `saveAsync` method, **ReadWriteItem** is the minimum permission level your app can specify in its manifest.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f97f9d1](https://github.com/OfficeDev/office-js-docs-pr/commits/f97f9d17ddac9354506521926b5635d419af0bc7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-token/docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910171829332808-1370%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=sQexOgHlWNCcayrJchX3AjGN6tFaEqBHPbpX1hNBM0A%3D&st=2019-10-17T18%3A25%3A43Z&se=2019-11-17T18%3A30%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by ElizabethSamuel-MSFT** [comment]
@kbrandl Thanks for reviewing and pointing out the table issue! Should all be sorted out now. Thx.

**Pull request merged**